### DISCUSSION TITLE
【0x09】Deploy your contract to polygon testnet & verify it on polygonscan

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#68] [Projects]
Link to L2 Lesson:
> https://github.com/WeLightProject/Web3-dApp-Camp/discussions/61

Video:

> https://www.youtube.com/watch?v=4Cvlj_r_YZU

Main Topic:
* Metamask 添加 Godwoken 网络，领取测试币
* 通过 Remix 在 Godwoken 部署合约
* 通过 Remix 在 Polygon Testnet 部署合约
* 合约的 Flattern 操作
* 在浏览器上验证（开源）合约

**Comment by alertcat** [comment]
![image](https://user-images.githubusercontent.com/73512451/163906058-70bb9dcf-671f-49a9-8f76-9db90fb709b1.png)

**Comment by yekai1003** [comment]
![image](https://user-images.githubusercontent.com/18655716/164387546-8df68352-82d4-4254-a13d-672bd1af0e00.png)
